doc/start/hardware-recommendations.rst

```diff
@@ -342,7 +348,7 @@ performance is measured as follows:
 Write Caches
 ------------
 
-Enterprise SSDs and HDDs normally include power loss protection features which
+Enterprise storage drives include power loss protection features which
 ensure data durability when power is lost while operating, and
 use multi-level caches to speed up direct or synchronous writes.  These devices
 can be toggled between two caching modes -- a volatile cache flushed to
```

Comment:
I see that this line was unmodified but the earlier m dash did not have spaces around it and I understand that'd be the preferred way. Could also go through these in a separate PR.